mopidy_soundcloud/soundcloud.py

```diff
@@ -348,14 +356,84 @@ def parse_track(self, data, remote_url=False):
 
         return Track(**track_kwargs)
 
+    def _update_public_client_id(self):
```

Comment:
What about an error/warning/debug(?) message at the end of the function stating a public client id couldn't be found (please correct me if that's not considered erroneous). I'd like it to be very obvious when the scraping needs fixing.

Reply:
Good one. I'll add a warning message.